aws-crt-http/aws-crt-http.csproj

```diff
@@ -1,9 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net35</TargetFrameworks>

```

Comment:
Could we also add `net45` while we're here, for the reference from the `net45` SDK?

Discussed with @normj yesterday - while we can technically reference the `net35` wrapper from the `net45` SDK that may be adding some complexity around the underling .NET runtime (2.0 vs 4.0), though can't point to a specific concern right now.

But hoping trivial to add given work already done to be compatible with `net35`.